main.go

```diff
@@ -33,40 +33,49 @@ func computeCLsVsPOI(bkg, sig, obs []float64) (POI, CLs_exp, CLs_obs []float64)
 
 	// Get B-only expectaction and associated toys
 	model_Bonly := modelPrediction(bkg, sig, 0.0)
-	pseudodata_Bonly := make([][]float64, 0)
-	for i := 0; i < Ntoys; i++ {
-		pseudodata_Bonly = append(pseudodata_Bonly, createPseudodata(model_Bonly))
+	pseudodata_Bonly := make([][]float64, Ntoys)
+	for i := range pseudodata_Bonly {
+		pseudodata_Bonly[i] = createPseudodata(model_Bonly)
 	}
 
 	// Start to loop over mu values
 	nPOI := 20
 	POI_start, POI_end := 0.0, 2.0
-	for iPOI := 0; iPOI < nPOI; iPOI++ {
+
+	POI = make([]float64, nPOI)
+	CLs_exp = make([]float64, nPOI)
+	CLs_obs = make([]float64, nPOI)
+
+	var (
```

Comment:
What is good practice here, about putting variable definition in `var( ... )` like here, or just one line per variable, like line 45-47 just above?

Reply:
that's a fine line b/w cosmetics (my own tack on what it should look like) and Go idiomactic code.
:)